packages/blade/src/components/Drawer/_decisions/decisions.md

```diff
@@ -0,0 +1,147 @@
+# Drawer
+
+A drawer is a panel that slides in mostly from right side of the screen over the existing content in the viewport. It helps in providing additional details or context and can also be used to promote product features or new products.
+
+<img width="500px" src="./2024-02-07-20-04-07.png" alt="Drawer Figma Skeleton">
+
+## Design
```

Comment:
Will Drawer be absolute positioned always? What if it needs to be used as a left/right navigation drawer?

Reply:
It will be fixed positioned on top right. It is always positioned like navigation drawer

Reply:
So we are setting a default position for Drawers on the top-right across all products? Like we set `bottom-left` for toasts?

Reply:
yup

Reply:
There are use-cases where drawer is used for navigation on left side in a lot of apps. Won't that ever be a use-case for us?

Reply:
Example: 
![image](https://github.com/razorpay/blade/assets/24487274/80176a81-e836-4a93-8751-3edd85117ac1)

Reply:
> There are use-cases where drawer is used for navigation on left side in a lot of apps. Won't that ever be a use-case for us?

Nope. So for desktop, our navigation bar itself is on left so there won't be anything opening from that side.

On mobile too we have drawers opening from right side only. Plus navigation component is something that we might tackle separately. This navigation component will use our Drawer internally on mobile. So at that time we can relook at this.

Anyway even if usecase comes up, its just addition of new prop so not much effort and no breaking change